src/main/frontend/modules/shortcut/dicts.cljc

```diff
@@ -80,16 +80,16 @@
    :editor/zoom-out                "Zoom out editing block / Backwards otherwise"
    :editor/toggle-undo-redo-mode   "Toggle undo redo mode (global or page only)"
    :editor/toggle-number-list      "Toggle number list"
-   :whiteboard/select              "Select tool"
```

Comment:
Would be useful to delete the other lang definitions for these shortcuts to keep translations accurate

Reply:
I reverted those changes. I was trying to explain the prefix and make it memorable, but that's not required anymore.